<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>780</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>文本</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>180</y>
        <width>421</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTElementUI::Text" name="defaultText">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Text" name="primaryText">
         <property name="text">
          <string>Primary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Text" name="successText">
         <property name="text">
          <string>Success</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Text" name="infoText">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Text" name="warningText">
         <property name="text">
          <string>Warning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Text" name="dangerText">
         <property name="text">
          <string>Danger</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>按钮</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>595</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTElementUI::Button" name="pushButton_1">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_2">
         <property name="text">
          <string>Primary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_3">
         <property name="text">
          <string>Success</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_4">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_5">
         <property name="text">
          <string>Warning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_6">
         <property name="text">
          <string>Danger</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>595</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTElementUI::Button" name="pushButton_7">
         <property name="text">
          <string>Plain</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_8">
         <property name="text">
          <string>Primary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_9">
         <property name="text">
          <string>Success</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_10">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_11">
         <property name="text">
          <string>Warning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_12">
         <property name="text">
          <string>Danger</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>595</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTElementUI::Button" name="pushButton_13">
         <property name="text">
          <string>Round</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_14">
         <property name="text">
          <string>Primary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_15">
         <property name="text">
          <string>Success</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_16">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_17">
         <property name="text">
          <string>Warning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_18">
         <property name="text">
          <string>Danger</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>283</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTElementUI::Button" name="pushButton_19">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_20">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_21">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_22">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_23">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_24">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTElementUI::Button" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>420</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTElementUI::Button" name="pushButton_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTElementUI::Button" name="pushButton_26">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTElementUI::Button" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTElementUI::Button" name="pushButton_29">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>420</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>595</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTElementUI::Button" name="pushButton_30">
         <property name="text">
          <string>Link</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_31">
         <property name="text">
          <string>Primary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_32">
         <property name="text">
          <string>Success</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_33">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_34">
         <property name="text">
          <string>Warning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_35">
         <property name="text">
          <string>Danger</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>570</y>
        <width>595</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTElementUI::Button" name="pushButton_36">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_37">
         <property name="text">
          <string>Primary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_38">
         <property name="text">
          <string>Success</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_39">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_40">
         <property name="text">
          <string>Warning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTElementUI::Button" name="pushButton_41">
         <property name="text">
          <string>Danger</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>滚动条</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>721</width>
        <height>551</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>719</width>
         <height>549</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>布局</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTElementUI::Text</class>
   <extends>QLabel</extends>
   <header>components/include/text.h</header>
  </customwidget>
  <customwidget>
   <class>QTElementUI::Button</class>
   <extends>QPushButton</extends>
   <header>components/include/button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
